sdk/healthinsights/azure-health-insights-cancerprofiling/src/main/java/com/azure/health/insights/cancerprofiling/AzureHealthInsightsServiceVersion.java

```diff
@@ -1,34 +1,40 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
 
 package com.azure.health.insights.cancerprofiling;
 
 import com.azure.core.util.ServiceVersion;
 
-/** Service version of CancerProfilingClient. */
-public enum CancerProfilingServiceVersion implements ServiceVersion {
-    /** Enum value 2023-03-01-preview. */
+/**
+ * Service version of AzureHealthInsightsClient.
+ */
+public enum AzureHealthInsightsServiceVersion implements ServiceVersion {
```

Comment:
This type name should not be changed.

Reply:
This was a change in the generated code, not sure how it had the old names.

Reply:
Will change in a follow-up PR